recipes/neo4j/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "neo4j" %}
```

Maybe this should be called `neo4j-python-driver`?
```suggestion
{% set name = "neo4j-python-driver" %}
```

---

I'm good with that.  I was following pypi but I agree with you that we shouldn't do that.